docs/text-area.md

```diff
@@ -0,0 +1,168 @@
+# Textarea
```

Comment:
Can you please add all the previews as landscaped images?

Reply:
added

Reply:
There seems to be a spelling mistake in the placeholder. @crstnmac